### PR TITLE
SEOImages: User Story 1463880, Part 3

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw Please review this PR. There are no warnings, all extensions are lowercase, and there are fewer files (26) than the previous PR :-).

**Comment by mairaw** [comment]
Resolved some merge conflicts

File: docs/framework/windows-workflow-foundation/performance.md [hunk 1]
```diff
@@ -355,19 +355,19 @@ public class Workflow1 : Activity
  The WF4 SQL persistence provider has tried to address some of these concerns.  The persistence tables expose certain information such as the active bookmarks and promotable properties.  The new content-based correlation feature in WF4 would not perform well using the WF3 SQL persistence approach, which has driven some change in the organization of the persisted workflow instance.  This makes the job of the persistence provider more complex and puts extra stress on the database.
 
 ### Environment Setup
- ![Workflow Performance Test Environment](./media/wfperfenvironment.gif "WFPerfEnvironment")
+
+![Workflow Performance Test Environment](./media/performance/performance-test-environment.gif "Performance test environment workflow")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion
![Environment setup for workflow performance test](./media/performance/performance-test-environment.gif)
```

File: docs/framework/windows-workflow-foundation/performance.md [hunk 2]
```diff
@@ -355,19 +355,19 @@ public class Workflow1 : Activity
...
 
 ### Test Setup
  Even with an improved feature set and better concurrency handling, the SQL persistence provider in WF4 is faster than the provider in WF3.  To showcase this, two workflows that perform essentially the same operations in WF3 and WF4 are compared below.
 
- ![Persistence Workflows](./media/persistworkflow.gif "PersistWorkflow")
-
- Figure 5 – Persistence workflow in WF3 on left and WF4 on right
+ ![Persistence Workflows](./media/performance/persist-workflow-in-wf3-and-wf4.gif "Persistence workflow in WF3 on left and WF4 on right")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion
 ![Persistence workflow in WF3 on left and WF4 on right](./media/performance/persist-workflow-in-wf3-and-wf4.gif "Persistence workflows")
```

File: docs/framework/windows-workflow-foundation/performance.md [hunk 3]
```diff
@@ -355,19 +355,19 @@ public class Workflow1 : Activity
...
 
  The two workflows are both created by a received message.  After sending an initial reply, the workflow is persisted.  In the WF3 case, an empty <xref:System.Workflow.ComponentModel.TransactionScopeActivity> is used to initiate the persistence.  The same could be achieved in WF3 by marking an activity as "persist on close."  A second, correlated message completes the workflow.  The workflows are persisted but not unloaded.
 
 ### Test Results
- ![Throughput Persistence](./media/throughputpersistence.gif "ThroughputPersistence")
+
+ ![Throughput Persistence](./media/performance/throughput-persistence-graph.gif "Throughput Persistence graph")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion
 ![Column chart showing throughput persistence](./media/performance/throughput-persistence-graph.gif "Throughput Persistence")
```

File: docs/framework/windows-workflow-foundation/performance.md [hunk 4]
```diff
@@ -423,21 +423,24 @@ public class Workflow1 : Activity
  Enabling workflow tracking will impact performance in varying degrees.  The benchmark below uses the logman tool to consume the ETW tracking events and record them to an ETL file.  The cost of the SQL tracking in AppFabric is not in the scope of this article.  The basic tracking profile, also used in AppFabric, is shown in this benchmark.  Also included is the cost of tracking only health monitoring events.  These events are useful for troubleshooting problems and determining the average throughput of the system.
 
 ### Environment Setup
- ![Workflow Performance Test Environment](./media/wfperfenvironment.gif "WFPerfEnvironment")
+
+ ![Workflow Performance Test Environment](./media/performance/performance-test-environment.gif "Performance test environment workflow")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion
 ![Environment setup for workflow performance test](./media/performance/performance-test-environment.gif)
```

File: docs/framework/windows-workflow-foundation/performance.md [hunk 5]
```diff
@@ -423,21 +423,24 @@ public class Workflow1 : Activity
...
 
 ### Test Results
- ![Workflow Tracing Cost](./media/workflowtracingcost.gif "WorkflowTracingCost")
+ ![Workflow Tracing Cost](./media/performance/workflow-tracking-costs.gif "Shows workflow tracking costs")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion

 ![Column chart showing workflow tracking costs](./media/performance/workflow-tracking-costs.gif "Workflow tracking cost")
```

File: docs/framework/windows-workflow-foundation/performance.md [hunk 6]
```diff
@@ -423,21 +423,24 @@ public class Workflow1 : Activity
...
 
  Health monitoring has roughly a 3% impact on throughput.  The basic profile’s cost is around 8%.
 
 ## Interop
  WF4 is almost a complete rewrite of [!INCLUDE[wf1](../../../includes/wf1-md.md)] and therefore WF3 workflows and activities are not directly compatible with WF4.  Many customers that adopted Windows Workflow Foundation early will have in-house or third-party workflow definitions and custom activities for WF3.  One way to ease the transition to WF4 is to use the Interop activity, which can execute WF3 activities from within a WF4 workflow.  It is recommended that the <xref:System.Activities.Statements.Interop> activity only be used when necessary. For more information about migrating to WF4 check out the [WF4 Migration Guidance](https://go.microsoft.com/fwlink/?LinkID=153313).
 
 ### Environment Setup
- ![Workflow Performance Test Environment](./media/wfperfenvironment.gif "WFPerfEnvironment")
+
+ ![Workflow Performance Test Environment](./media/performance/performance-test-environment.gif "Performance test environment workflow")
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/performance.md]:
```suggestion
 ![Environment setup for workflow performance test](./media/performance/performance-test-environment.gif)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @damabe. Left some comments regarding the changes. I'd focus more on adding the text to the alt text vs. title. I'm also suggesting a few unrelated changes if you don't mind.

**Comment by damabe** [comment]
@mairaw The feedback has been incorporated. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031200050/Report/. There are three failures in alt text length. Do you want that criteria included in this user story or fixed at a later time?

**Comment by mairaw** [comment]
What does work best for you and your team @damabe? I'm fine either way.

**Review by mairaw — APPROVED** [review]
I'm approving and merging as-is. Changes look great @damabe. Let me know how you prefer to handle these so I know what to review in the following PRs.

**Pull request merged**